.github/ISSUE_TEMPLATE/help_desk.md

```diff
@@ -0,0 +1,41 @@
+---
+name: Help Desk
+about: Despite multiple attempts you repeatedly fail to solve an issue?  
+title: "[Help Desk]"
+labels: help desk
+assignees: ''
+
+---
+ 
+## Please note
+<!-- A brief description of what you expected to happen. -->
+- We may ignore any issues due to a lack of time and apologize.
+- Please first always refer to the [documentation](https://k-wave-python.readthedocs.io/en/latest/index.html), or specific [examples](https://github.com/waltsims/k-wave-python/tree/master/examples) before raising an issue
+- Code dumps "can you help, here is my code" will be closed without comment 
+- Be as specific as possible and write in detail what you have already tried, add links and any reference that may help us understand 
+- Run your simulation with verbosity level set to two
+```python
+execution_options = SimulationExecutionOptions(is_gpu_simulation=True, verbose_level=2)
+```
+
+## Environment information
```

Comment:
Perhaps you can add the commands they can run to get this information? They do this on scipy for example.

Reply:
We might be able to query many global variables in kwave.__init__